docs/source/conceptual_guides/adapter.md

```diff
@@ -0,0 +1,34 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Adapters
```

Comment:
I wonder if we could cause confusion here. We use the name "adapter" pretty liberally throughout PEFT for all kinds of methods. Users may, e.g., pass an `adapter_name` to a LoRA model. This section makes it seem that LLaMA adapter is the the only "adapter" method in PEFT.

Honestly, I'm not sure how to correctly categorize this method, there is no clear _taxonomy_ of adapter/tuner methods :-/ But I'd rather combine this with `lora.md` and call all of them adapter, as "reparametrization" is not a name I have seen used colloquially for these methods.

Reply:
I was struggling a bit with the taxonomy of it all as well, but I think your approach is better for avoiding confusion! 👍